### ISSUE TITLE
Separate services

### ISSUE THREAD

**Issue by godsgood33** [twofas/2fas-server#8]
Is there a way that I can change the docker-compose to use an already existing mysql and redis services?  They are running on different hardware so the system would have to address IPs and not just internal docker naming.  Thanks!

**Comment by ripaire** [comment]
Can you please guide me to setup 2fas on my server ?

**Comment by Potherca** [comment]
> Is there a way that I can change the docker-compose to use an already existing mysql and redis services?

Yes. Although I do feel this is a more generic DevOps question and less useful to ask here...

To use another MySQL, remove the `mysql` entry from the docker-compose file entirely, and change the config in [`.env`](https://github.com/twofas/2fas-server/blob/main/.env) to point to the existing DB instance. 

For Redis, remove the `redis` entry in the docker-compose file and change the `service_url` for the Redis entry in [config.yml](https://github.com/twofas/2fas-server/blob/main/config/config.yml) to meet your needs.
